.github/workflows/bump_version.yml

```diff
@@ -0,0 +1,60 @@
+name: Bump Version
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version (optional)'
+        required: false
+        default: ''
+  
+jobs:
+  build:
+    name: Bump Version
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install jq
+      run: |
+        set -e
+        sudo apt update
+        sudo apt-get install -y jq
+
+    - name: ⇣ Checkout
+      uses: actions/checkout@v2
+      with:
+        token: ${{ secrets.GH_BUILDER_TOKEN }}
+        fetch-depth: 0
+
+    - name: Set version
+        run: |
+          set -ex
+          OLD_VERSION="$(cat package.json | jq '.version' | tr -d '\\"')"
+          NEW_VERSION="${{ github.event.inputs.version }}"
+          if [ -z "$NEW_VERSION" ]; then
+            NEW_VERSION=$(echo $OLD_VERSION | awk -F. '{$NF = $NF + 1;} 1' | sed 's/ /./g' )
+          fi
+          NEW_VERSION=$NEW_VERSION jq '.version=env.NEW_VERSION' package.json > /tmp/package.json
+          mv /tmp/package.json package.json
+    
+    - name: Get Version
+      id: get_build_version
+      run: |
+        set -ex
+        VERSION="$(cat package.json | jq '.version' | tr -d '\\"')"
```

Comment:
now that i updated the package.json file, I'm acquiring the new version the same way I acquired `OLD_VERSION` in the previous step - directly from the package.json file